system/Common.php

```diff
@@ -203,7 +204,12 @@ function command(string $command)
     /**
      * More simple way of getting config instances from Factories
      *
-     * @return object|null
+     * @template ConfigTemplate of BaseConfig
+     *
+     * @param class-string<ConfigTemplate>|string $name
+     *
+     * @return ConfigTemplate
+     * @phpstan-return ($name is class-string<ConfigTemplate> ? ConfigTemplate : object|null)
```

Comment:
Does it really return a generic object? I thought if not found the function will return `null`.

Reply:
I add null as a possible return